assets/snippets/binary-databases.md

```diff
@@ -1,7 +1,10 @@
 Binary databases make use of the
 [MaxMind DB file format](https://maxmind.github.io/MaxMind-DB/). MaxMind
-provides official client APIs in the language listed below. The following client
-APIs are open source and licensed under the Apache License, Version 2.0.
+provides [official client APIs](https://dev.maxmind.com/geoip/docs/databases/#official-client-apis)
```

Comment:
These links probably shouldn't include the scheme and hostname. It might make sense to use `ref`/`relref` too, but I don't see us using that Hugo feature elsewhere.

Reply:
Good catch: https://github.com/maxmind/dev-site/pull/1243/commits/9dd2de3f2e9eaa58f45a9fbda1ebfa0b741c7230